testing/tests/DevExpress.ui.widgets/overlay.tests.js

```diff
@@ -496,6 +496,12 @@ testModule('option', moduleConfig, () => {
 
             assert.strictEqual(this.$wrapper.attr('someAttr'), 'someValue');
         });
+
+        test('does not override old clases', function(assert) {
```

Comment:
I don't see the way `_renderWrapperAttributes` call removes default classes from the wrapper. 

Moreover, a lot of another tests will be failed if this happens

Reply:
attr() overrides all classes, thats why we need use addClass() instead